extra/resources/HealthSMART.in

```diff
@@ -308,6 +309,20 @@ HealthSMART_validate() {
     return $OCF_SUCCESS
 }
 
+HealthSMART_reload_agent() {
+    state_dir=$(dirname "$OCF_RESKEY_state")
+    if [ -f "$state_dir/$$" ]; then
+        rm "$state_dir/$$"
+    fi
+
+    touch "$state_dir/$$"
+    if [ $? -ne 0 ]; then
+        return $OCF_ERR_ARGS
+    fi
+
+    return $OCF_SUCCESS
```

Comment:
The above line is all that reload needs to do

Since state is not reloadable, reload will not be called if it changes

Reply:
Fixed.